packages/flutter_tools/test/general.shard/dart/pub_get_test.dart

```diff
@@ -1008,48 +988,16 @@ exit code: 66
     expect(processManager, hasNoRemainingExpectations);
   });
 
-  testWithoutContext('analytics sent on success', () async {
```

Comment:
There is no GA4 `Event` defined for recording whether or not pub succeeds or fails.